.github/workflows/ruby-ci.yml

```
@@ -0,0 +1,52 @@
+name: Ruby Testing
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:12.1
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

I think the env variables should be on the top of the file. e.g:

```suggestion
name: Ruby Testing
on: [pull_request]
env:
  RAILS_ENV: test
  DATABASE_URL: postgresql://postgres:@localhost/test
  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
jobs:
  test:
    runs-on: ubuntu-latest
    services:
        postgres:
            image: postgres:12.1
            ports: ['5432:5432']
            options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```